app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -129,6 +138,7 @@
     private DescriptionsAdapter descriptionsAdapter;
     private RVRendererAdapter<CategoryItem> categoriesAdapter;
     private ProgressDialog progressDialog;
+    private int flag = 0, s_count = 0;
```

Comment:
@nfatma What are these variables for? Can you rename it to be more descriptive?

Reply:
yes, alright:)